lib/service/Dialog.dart

```diff
@@ -268,7 +266,7 @@ Dialog walletDialog(BuildContext context) {
             ],
           ),
           Visibility(
-            visible: isWeb,
+            visible: (kIsWeb)? false : true,
```

Comment:
You can simplify this to `visible: !kIsWeb`.

Reply:
done